modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -123,6 +129,46 @@ Number of Kafka RPC service errors.
 
 ---
 
+=== vectorized_ntp_archiver_compacted_replaced_bytes
+
+Number of bytes removed from cloud storage by compaction operations. This is tracked on a per-topic and per-partition basis.
+
+This metric resets every time partition leadership changes. It primarily exists to track the fact that compaction is performing operations on cloud storage.
+
+The `namespace` label supports the following options for this metric:
+
+* `kafka` - user topics
+* `kafka_internal` - internal kafka topic such as consumer groups
+* `redpanda - redpanda-only internal data
+
+*Labels*:
+
+* `namespace=("kafka" | "kafka_internal" | "redpanda")`
+* `topic`
+* `partition`
+
+---
+
+=== vectorized_ntp_archiver_pending
+
+The offset of the last committed offset uploaded to tiered storage for each partition. This metric matching the last committed offset of a partition indicates that all data for a partition is uploaded to tiered storage.
+
+This metric is impacted by the `cloud_storage_upload_loop_max_backoff_ms` configuration property. If this metrics lags behind the latest committed offset by more than twice this property value, it may indicate a network issue in the upload path for the partition, or it may indicate a cluster that is under too much pressure.
```

Comment:
```suggestion
This metric is impacted by the `cloud_storage_upload_loop_max_backoff_ms` configuration property. If `vectorized_ntp_archiver_pending` lags behind the latest committed offset by more than twice this property value, it may indicate a network issue in the upload path for the partition, or it may indicate a cluster that is under too much pressure.
```